### PR TITLE
add handling for 802.1Q I-Tags, used for Provider Backbone Bridging

### PR DESCRIPTION
**Pull request by dgwynne**
This is just a checklist to guide you. You can remove it safely.

**Checklist:**

-   [ ] If you are new to Scapy: I have checked <https://github.com/secdev/scapy/blob/master/CONTRIBUTING.md> (esp. section submitting-pull-requests)
-   [ ] I squashed commits belonging together
-   [ ] I added unit tests or explained why they are not relevant
-   [ ] I executed the regression tests for Python2 and Python3 (using `tox` or, `cd test && ./run_tests_py2, cd test && ./run_tests_py3`)

> brief description what this PR will do, e.g. fixes broken dissection of XXX

> if required - short explanation why you fixed something in a way that may look more complicated as it actually is

> if required - outline impacts on other parts of the library

fixes #xxx (add issue number here if appropriate, else remove this line)

### CODE REVIEW

**Comment by guedou** [comment]
Thanks for this PR. Could you add some unit tests?

**Comment by dgwynne** [comment]
If you point me at one that you think is comparable, I'll try to figure out a unit test.

**Comment by p-l-** [comment]
Also, could you add (e.g., as a comment) a reference to the protocol this PR implements?

**Comment by guedou** [comment]
Building and dissecting a packet will be perfect. You can find an example here https://github.com/secdev/scapy/blob/master/test/regression.uts#L9061

You can add similar tests for `Dot1QITag` under the `L2TP` ones.

**Comment by gpotter2** [comment]
It appears we already have support for this layer:

https://github.com/secdev/scapy/blob/master/scapy/contrib/spbm.py

But you may improve it. Feel free to add the missing `bind_layers` to that file. You may load the module through `load_contrib("spbm")` or `from scapy.arch.contrib.spbm import *`

File: scapy/layers/l2.py [hunk 1]
```diff
@@ -285,6 +285,23 @@ def mysummary(self):
...
+class Dot1QITag(Packet):
+    name = "802.1Q I-Tag"
+    aliastypes = [Ether]
+    fields_desc = [BitField("prio", 0, 3),
+                   BitField("dei", 0, 1),
+                   BitField("uca", 0, 1),
+                   BitField("res", 0, 3),
+                   BitField("sid", 0, 24)]
+
+    def default_payload_class(self, pay):
```
Comment (p-l-) [inline scapy/layers/l2.py]: This is usually achieved using a `bind_layers()` call.

**Pull request closed without merging**